homeassistant/components/lamarzocco/binary_sensor.py

```diff
@@ -57,6 +58,16 @@ class LaMarzoccoBinarySensorEntityDescription(
     ),
 )
 
+SCALE_ENTITIES: tuple[LaMarzoccoBinarySensorEntityDescription, ...] = (
+    LaMarzoccoBinarySensorEntityDescription(
+        key="connected",
+        translation_key=None,
+        device_class=BinarySensorDeviceClass.CONNECTIVITY,
+        is_on_fn=lambda config: config.scale.connected if config.scale else False,
```

Comment:
if there is no scale, shouldn't it be unknown?

Reply:
Theoretical question. If there is no scale this doesn't exist and I didn't want to change return type.